QUIZ/quiz1-program8.ipynb

```diff
@@ -0,0 +1,99 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 41,
+   "id": "d3153549",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import csv"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 42,
+   "id": "eab9f4d6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "header=[\"name\",\"contact\"]\n",
+    "\n",
+    "data = [[\"om\", \"9316203288\"],\n",
+    "        [\"sai\",\"8767564567\"],\n",
+    "        [\"ram\", \"8967567890\"],\n",
+    "        [\"radhe\", \"9067567890\"],\n",
+    "        [\"krishna\", \"9167567890\"]]\n",
+    "filename = \"Contacts.csv\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 43,
+   "id": "cf6e292c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with open('filename','w',newline='') as f:\n",
+    "    k=csv.writer(f,delimiter='|')\n",
+    "    k.writerow(header)\n",
+    "    k.writerows(data)"
+   ]
```

Comment:
writerows() and writerow() ...any difference?

Reply:
writerows() function is used to print multiple rows. AND
writerow() function is used to print only one row.